.github/workflows/integration-tests.yml

```
@@ -51,23 +51,26 @@ jobs:
       - name: Run Python doc tests
         run: |
           python -m doctest qdrant_client/local/local_collection.py
       - name: Run integration tests
+        env:
+          HF_TOKEN={{ secrets.HF_TOKEN }}
         run: |
           ./tests/integration-tests.sh
         shell: bash
       - name: Backward compatibility integration tests
+        env:
+          HF_TOKEN={{ secrets.HF_TOKEN }}
```

actionlint (1.7.4)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)